gunicorn/glogging.py

```diff
@@ -195,8 +198,16 @@ def setup(self, cfg):
         self.access_log.setLevel(logging.INFO)
 
         # set gunicorn.error handler
+        if self.cfg.capture_stdio and cfg.errorlog != "-":
+            for stream in sys.stdout, sys.stderr:
+                stream.flush()
+
+            self.logfile = open(cfg.errorlog, 'a+')
```

Comment:
hrm not sure about it, it's more readable that way. But is there a rule somewhere about it?